documents/markdown/resolution_process.md

```diff
@@ -0,0 +1,96 @@
+# The Open Permissions Platform Resolution Workflow
```

Comment:
So a diagram (flow chart/ activity diagram) would help. Also rename this as LLD I think.